hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1374,6 +1374,14 @@ private void setWriteTimer(HoodieTable table) {
     }
   }
 
+  public void setWriteTimer(String commitType) {
+    if (commitType.equals(HoodieTimeline.COMMIT_ACTION)) {
```

Comment:
We can remove this method: `void setWriteTimer(HoodieTable table)`

Reply:
@danny0405 ok, done